Userland/Libraries/LibGUI/Event.h

```diff
@@ -57,13 +57,15 @@ class Event : public Core::Event {
         ActionEnter,
         ActionLeave,
         AppletAreaRectChange,
+        AppletVerticalChange,
 
         __Begin_WM_Events,
         WM_WindowRemoved,
         WM_WindowStateChanged,
         WM_WindowRectChanged,
         WM_WindowIconBitmapChanged,
         WM_AppletAreaSizeChanged,
+        WM_AppletVerticalChanged,
```

Comment:
What does `AppletVerticalChanged` mean? A better name could be `AppletOrientationChanged`.